kolibri/core/apps.py

```diff
@@ -124,13 +119,29 @@ def check_redis_settings():
                     "Kolibri, the following is suggested: maxmemory-policy allkeys-lru"
                 )
 
-            # should only change something when `config_use_conf` is also `False`
-            if helper.changed:
-                helper.save()
-            elif maxmemory == 0 or maxmemory_policy == "noeviction":
+            # default setting is "0", or no limit
+            maxmemory = helper.get_maxmemory()
+            if config_maxmemory > 0 and maxmemory != config_maxmemory:
+                used_memory = helper.get_used_memory()
+                if maxmemory < used_memory:
```

Comment:
should not this be `config_maxmemory` ?

Reply:
Ah yes, not sure how I didn't catch that. Thanks!